geomstats/geometry/invariant_metric.py

```diff
@@ -778,21 +745,28 @@ def parallel_transport(
 
         def acceleration(state, time):
             """Compute the right-hand-side of the parallel transport eq."""
-            omega, zeta = state[1:]
-            gam_dot, omega_dot = self.geodesic_equation(state[:2], time)
+            omega = state[..., 1, :, :]
```

Comment:
omega and zeta are not very meaningful: any chance we could rename these variables by the concepts they represent (not by a greek letter found in one paper)?

Reply:
Same as above.